javascript-modal/styles.css

```diff
@@ -0,0 +1,82 @@
+body {
+  width: 960px;
+  font-family: helvetica, arial, sans-serif;
+}
+
+.button {
+  background-color: blue;
+  color: white;
+  width: 120px;
+  padding: 7px 20px 7px 20px;
+}
+
+.buttonPosition {
+  position: relative;
+}
+
+.button:hover {
+  cursor: pointer;
+}
+
+.button:active {
+  top: 3px;
+}
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 400px;
+  padding: 25px 10px 30px 10px;
+  text-align: center;
+  margin-top: 15%;
+  margin-left: 30%;
+  margin-right: 30%;
```

Comment:
This is a lot of manual movement with margin. You can avoid this if you make the modal a child of the overlay element, and make the overlay a flex container with its content centered vertically and horizontally.